<?xml version="1.0" ?>

<launch>

    <arg name="robot_ip" default="130.230.37.115"/>
    <arg name="load_gripper" default="true" />
    <include file="$(find panda_moveit_config)/launch/panda_control_moveit_rviz.launch" >
      <arg name="robot_ip" value="$(arg robot_ip)" />
      <arg name="load_gripper" value="$(arg load_gripper)" />
    </include>

    <include file="$(find realsense2_camera)/launch/rs_camera.launch" />

    <include file="$(find apriltag_ros)/launch/continuous_detection.launch" />

    <node pkg="cobot_vision" type="calibration.py" name="tf_broadcaster" output="screen" />


</launch>
